torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -205,6 +215,48 @@ void InplaceMKLDNNSubgraph(std::shared_ptr<Graph> graph) {
   }
 }
 
+// aten_op is a sequence of aten ops that implement a required operation
+// given input output tensors
+Operation createUnaryOp(
+    std::function<void(at::Tensor output, at::Tensor input)> aten_op,
+    bool inplace = false) {
+  return [aten_op, inplace](Stack* stack) {
+    auto a = pop(stack).toTensor();
+    c10::impl::ExcludeDispatchKeyGuard edkg(c10::autograd_dispatch_keyset);
```

Comment:
Can you add a description at the top of this about mkldnn padded tensors, the constraint of `eltwise_ops(0) = 0`, and what we are about to do ?